docs/compose/RatingBar/README.md

```diff
@@ -21,34 +21,11 @@ Example of a rating bar:
 ```Kotlin
 import net.skyscanner.backpack.compose.ratingbar.BpkRatingBar
 
-BpkRatingBar(
```

Comment:
just simplified the readme as it's a very simple component, so we can just show all props in one example